### PR TITLE
EICNET-1215: Update email notification texts

### PR DESCRIPTION
**Pull request by 4hardfire**
### Updated message templates:

- NOTIFY :: MT01 :: Group request submitted -> https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EICNET-2019
- NOTIFY :: MT02 :: Group request approved -> https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EICNET-2021
- NOTIFY :: MT03 :: Welcome user to group -> https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EICNET-2026
- NOTIFY :: MT04 :: Membership request rejected -> https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EICNET-2027
- NOTIFY :: MT08 :: Group access change -> https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EICNET-2028
- NOTIFY :: MT11 :: Group membership requested (to requester) -> https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EICNET-2025 (**The logic to trigger this notification has not been yet implemented**)
- NOTIFY :: MT12 :: Group requested (to requester) -> https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EICNET-2020
- NOTIFY :: MT13 :: New member joined -> https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EICNET-2023
- NOTIFY :: MT14 :: New membership request -> https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EICNET-2024
- NOTIFY :: MT :: Block request - request accepted -> https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EICNET-2032
- NOTIFY :: MT31 :: Group Re-published -> https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EICNET-2033 (Created new message and update triggers).

### Tests

**NOTIFY :: MT01 :: Group request submitted** and **NOTIFY :: MT12 :: Group requested (to requester)**
- [x] As TU, create a new group
- [x] As SA/SCM, run cron
- [x] Make sure the TU receives a notification (**MT12**) and the text is according to the FA
- [x] Make sure all SA/SCM users receives a notification (**MT01**) and the text is according to the FA

**NOTIFY :: MT02 :: Group request approved**
- [x] As SA/SCM, change a PENDING group to DRAFT
- [x] Run cron
- [x] Make sure the GO receives a notification and the text is according to the FA

**NOTIFY :: MT03 :: Welcome user to group**
- [x] As TU, go to a public group where joining method is set to "Request"
- [x] Request membership
- [x] As GO accept the membership
- [x] Run cron
- [x] TU should get a welcome e-mail

**NOTIFY :: MT04 :: Membership request rejected**
- [x] As TU, go to a public group where joining method is set to "Request"
- [x] Request membership
- [x] As GO deny the membership
- [x] Run cron
- [x] TU should get a notification about the request deny and the text is according to the FA.

**NOTIFY :: MT08 :: Group access change**
- [x] As SA/SCM, edit a public group and change its visibility to private
- [x] Run cron
- [x] Make sure the GO and all GAs receive a notification and the text is according to the FA

**NOTIFY :: MT13 :: New member joined**
- [x] As TU, join a public group
- [x] Run cron
- [x] Make sure the GO receives a notification and the text is according to the FA

**NOTIFY :: MT14 :: New membership request**
- [x] As TU, go to a public group where the joining method is set to "Request to join"
- [x] Request to join the public group
- [x] Run cron
- [x] Make sure the GO receives a notification and the text is according to the FA

**NOTIFY :: MT :: Block request - request accepted**
- [x] As SA/SCM, block a group and provide a reason
- [x] Run cron
- [x] Make sure the GO receives a notification and the text is according to the FA

**NOTIFY :: MT31 :: Group Re-published**
- [x] As SA/SCM, unblock a group (change status from blocked to published)
- [x] Run cron
- [x] Make sure the GO receives a notification about the group being re-published and the text is according to the FA
- [x] As SA/SCM, un-archive a group (change status from archived to published)
- [x] Run cron
- [x] Make sure the GO receives a notification about the group being re-published and the text is according to the FA

### CODE REVIEW

File: config/sync/language/en/message.template.notify_welcome_user_to_group.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
-    format: plain_text
+    value: "<p>Dear [message:author:display-name],</p>\r\n\r\n<p>Thank you for your interest in the [message:field_group_ref:entity:type-name]&nbsp;<a href=\"[message:field_group_ref:entity:url]\">[message:field_group_ref:entity:title]</a>.</p>\r\n\r\n<p>Your membership request has been approved. You can now access&nbsp;<a href=\"[message:field_group_ref:entity:url]\">[message:field_group_ref:entity:title]</a>.</p>\r\n\r\n<p>Don't hesitate to&nbsp;<a href=\"[message:field_group_ref:entity:author:url]contact\"><u>contact me</u></a>&nbsp;if you have further questions.</p>\r\n\r\n<p>[message:field_group_ref:entity:author:display-name],<br />\r\nGroup Administrator - [message:field_group_ref:entity:title]</p>\r\n"
```
Comment (lramarojaona) [inline config/sync/language/en/message.template.notify_welcome_user_to_group.yml:6]: @4hardfire , here the link to contact is wrong, it's missing a slash

File: config/sync/message.template.notify_group_access_change.yml [hunk 2]
```diff
@@ -14,14 +15,14 @@ label: 'NOTIFY :: MT08 :: Group access change'
...
-    format: plain_text
+    value: "<p>Dear member,<br />\r\n<br />\r\nThe [message:field_group_ref:entity:type-name] visibility settings have been updated to [message:field_group_visibility:value].<br />\r\n<br />\r\nPlease consult the [message:field_group_ref:entity:type-name] <a href=\"[message:field_group_ref:entity:url]/about#featured-content-section-admin\">about page</a>&nbsp;for further details.<br />\r\n<br />\r\nPlease contact the [message:field_group_ref:entity:type-name] owner: [message:field_group_ref:entity:author:display-name] in case of questions.<br />\r\n<br />\r\n[message:field_group_ref:entity:author:display-name]<br />\r\n[message:field_group_ref:entity:type-name] Administrator - [message:field_group_ref:entity:title].</p>\r\n"
```
Comment (lramarojaona) [inline config/sync/message.template.notify_group_access_change.yml:21]: Here I added a comment to the ticket.
Your implentation is correct regarding the FA, but the FA shouyld be modified IMO.
https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EICNET-2028?focusedCommentId=5343075&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-5343075

**Review by lramarojaona — APPROVED** [review]

File: config/sync/language/en/message.template.notify_welcome_user_to_group.yml (hunk 1 above)
Comment (4hardfire) [reply config/sync/language/en/message.template.notify_welcome_user_to_group.yml]: Done!

File: config/sync/message.template.notify_group_access_change.yml (hunk 2 above)
Comment (4hardfire) [reply config/sync/message.template.notify_group_access_change.yml:21]: I put the ticket in blocked since we need to discuss.

**Pull request merged**